tools/node_compat/TODO.md

```diff
@@ -3,7 +3,7 @@
 
 NOTE: This file should not be manually edited. Please edit `tests/node_compat/config.json` and run `deno task setup` in `tools/node_compat` dir instead.
 
-Total: 2998
+Total: 2997
```

Comment:
this value in `TODO.md` is read from the scripts as part of the tests?

Reply:
`tools/node_compat/setup.ts --check` is run in CI. I think we landed two PRs after one another that enabled some tests and they both changed the value from 2999 to 2998.